http/handler.go

```diff
@@ -117,6 +117,18 @@ func NewHandler(opts ...HandlerOption) (*Handler, error) {
 	return handler, nil
 }
 
+func (h *Handler) Serve(ln net.Listener, closing <-chan struct{}) {
```

Comment:
Same here, even though this is the implementation of an interface, we should have doc strings.